packages/contracts/test/JwtVerifier/JwtVerifier.t.sol

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.12;
+
+import "forge-std/Test.sol";
+import "forge-std/console.sol";
+import {EmailProof, JwtVerifier} from "../../src/utils/JwtVerifier.sol";
+import {JwtGroth16Verifier} from "../../src/utils/JwtGroth16Verifier.sol";
+import {ERC1967Proxy} from "@openzeppelin/contracts/proxy/ERC1967/ERC1967Proxy.sol";
+
+contract JwtVerifierTest_verifyEmailProof is Test {
+    JwtVerifier verifier;
+
+    constructor() {}
+
+    function setUp() public {
+        JwtVerifier verifierImpl = new JwtVerifier();
+        JwtGroth16Verifier groth16Verifier = new JwtGroth16Verifier();
+        ERC1967Proxy verifierProxy = new ERC1967Proxy(
+            address(verifierImpl),
+            abi.encodeCall(
+                verifierImpl.initialize,
+                (msg.sender, address(groth16Verifier))
+            )
+        );
+        verifier = JwtVerifier(address(verifierProxy));
+    }
+
+    function test_verifyEmailProof() public {
```

Comment:
@wshino Would love to see an actual integration test here.

Reply:
@shreyas-londhe I implemented